frontend/src/routes/explore/+page.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+    import data from "$lib/data";
+</script>
+
+<!-- this fixes the issue of weird extra space to the right of the page -->
+<div
+    class="columns my-0"
+    style="width: calc(100vw + 12px); height: calc(100vh - 56px);"
+>
+    <div class="column is-one-quarter has-background-grey-darker">
```

Comment:
My best shot at it: 
```html
<div
        class="column is-one-quarter has-background-grey-darker"
        style:padding-left="24px">
        <div class="level my-2">
            <div class="level-left mx-4">
                <div class="level-item">
                    <h1 class="title is-5">Watchlist</h1>
                </div>
            </div>

            <div class="level-right">
                <div class="level-item">
                    <span class="icon">
                        <a href="/"><i class="fas fa-plus" /></a>
                    </span>
                </div>
            </div>
        </div>
        <hr />
        <div class="mx-4">
            <p>Hi!</p>
        </div>
    </div>
```

Reply:
Good idea to include a line break :)
I'll take that into account and throw on a table on the next commit